app/main.py

```diff
@@ -1,3 +1,44 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> (int, float):
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float)) -> (int, float):
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        self.km = self.km + other
+        return self
+
+    def __iadd__(self, other: (int, float)) -> (int, float):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, other: (int, float)) -> float:
+        return Distance(km=round(self.km * other, 3))
+
+    def __truediv__(self, other: (int, float)) -> float:
+        return Distance(km=round(self.km / other, 2))
+
+    def __lt__(self, other: (int, float)) -> bool:
+        return self.km < other
```

Comment:
what if `other` has a `Distance` type? You should add a checking of the `other` type